### DISCUSSION TITLE
v1.3.0 - Resource Insight Improvements and More!

### DISCUSSION THREAD

**Discussion by aquasarus** [aquasarus/Starfield-Interactive-Smart-Slate#20] [Announcements]
v1.3.0 adds more insights to the data you've collected as an explorer. You're now able to look up multiple resources simultaneously to find the perfect planet to place your next outpost. Lifeforms can now be marked as "outpost production allowed" for those who want to domesticate aliens. This update also marks the beginning of our localization efforts, with French being the first milestone.

## Installation
Head to the [release page](https://github.com/aquasarus/Starfield-Interactive-Smart-Slate/releases) to download this. Simply extract and use! Your data is stored in a [separate folder](https://github.com/aquasarus/Starfield-Interactive-Smart-Slate/wiki/User-Data), and so will not be wiped.

#### Multiple Resource Search
You can now hold `CTRL` to multi-select in the resource search tabs. This allows you to pinpoint planets where one or more resources are available simultaneously.

 <img src="https://raw.githubusercontent.com/aquasarus/Starfield-Interactive-Smart-Slate/main/Gifs/multiple_resource_search.gif" width=70% height=70%>

#### The "Farmable" Attribute
A subset of the fauna/flora in-game can be domesticated. Now you can store this information through a checkbox, and it will reflect in the ISS through a new simple icon (🐄 and 🥕).

 <img src="https://raw.githubusercontent.com/aquasarus/Starfield-Interactive-Smart-Slate/main/Gifs/farmable_attribute.gif" width=50% height=50%>

#### French Autocomplete
Since Starfield doesn't let us pull data directly from the game yet, manual entry is still a big part of the ISS. Making it easy to enter fauna/flora names is pretty important. You can now choose French in the settings tab, allowing you to get autocomplete based on the French version of Starfield.

 <img src="https://raw.githubusercontent.com/aquasarus/Starfield-Interactive-Smart-Slate/main/Gifs/french_autocomplete.gif" width=40% height=40%>

#### Unlocking Lifeform Counts
People are already making mods that add more lifeforms to the game. This will change the total fauna/flora count on some planets and moons. You now have the option to unlock this in the ISS in order to add as many fauna and flora as you'd like. With this change, we've also added the option to delete them (in the same way as outposts).
- P.S. All deletion in the app is reversible. Check out [the wiki](https://github.com/aquasarus/Starfield-Interactive-Smart-Slate/wiki/Outposts) to find out how.

## Wanna Help?
This update cost 37 hours of my free time. The most rewarding part of this is knowing that someone is out there enjoying the app! Here are some ways you can help out if you're interested:
- My game pass subscription has expired, so it'll be harder for me to continue working on this app without owning a copy of Starfield. Message me if you want to gift me a copy to help fund the efforts!
- Want autocomplete for other languages? [Open an issue](https://github.com/aquasarus/Starfield-Interactive-Smart-Slate/issues) and we can work together to find the corresponding fauna/flora names in your language!

## Future Plans
Part of the changes in v1.3.0 is actually a large refactor to lay the groundwork for more collaboration. We're slowly working on a complete UI revamp that will make the ISS look a lot better, and a lot closer to the in-game UI styles. It will take time, but it's coming together. Here are some other things that I'd like to add in the future:
- Extending current pictures/notes functionality to planets and moons.
- Easier navigation from resource search results back to the main tab.
- A new stats/summary tab for exploration progress tracking.

## Credits
Credit where credit is due. Some of you have directly or indirectly contributed to the development of **v1.3.0**. A heartfelt thank you is in order!

- @RSchwoerer
- @MaamuT
- @Zatopek33
- [voal](https://www.nexusmods.com/starfield/users/9545871)

**Comment by DeHades666** [comment]
Great update, thanks =) Multiple resources and Outpost Production allowed, thought that was possibly a daft request when I made it, but extremely useful =) Not sure how much Game pass is or indeed how much the game costs (was bought for me as a gift), and can't promise a lot, but can certainly send a few bucks/pounds sterling your way as a contribution towards it

**Reply by aquasarus** [reply]
Hey! Glad you're still using the app. Those features were honestly very easy to add. Life just took my time away from working on this. And don't worry about tipping me haha. If the cost of this game is anything more than negligible for you, then I wouldn't want you to bother anyway.
